packages/sanity/src/core/releases/store/types.ts

```diff
@@ -23,6 +23,7 @@ export type ReleaseFinalDocumentState = {
 }
 
 /**
+ * TODO: When made `beta`, update the PublishDocumentVersionEvent to use this type
```

Comment:
Has this been noted somewhere like a linear story? Or is it being made as part of a fast follow up PR?

Reply:
I think it's probably enough to have it here given that the change to `@beta` will have to be done also here by updating the release tag of this ReleaseDocument interface